src/global.scss

```diff
@@ -59,10 +59,42 @@
 }
 
 .content-wrapper {
+    display: inline;
     &:focus {
         outline: none;
         box-shadow: none;
     }
+
+    // Depending on the content type and whether the above Decorator has
+    // breadcrumbs or language selector (språkvelger), the
+    // content beneath has to be adjusted for spacing in order to
+    // give a balanced look & feel. (Ie. "some white space, but not too much").
+    // The following :before element will ajust according to which
+    // classes is set in PageWrapper.
+    &:before {
+        content: '';
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        padding: var(--a-spacing-4) 0;
+        background-color: transparent;
+
+        @include common.full-width-mixin();
+    }
```

Comment:
Kunne dette vært løst litt enklere, kanskje med en padding på app containeren?

Reply:
Grunnen til at jeg valgte before-psudoen er for å kunne sette bakgrunnsfarge uten å påvirke hele wrapperen. Feks nyheter skal kunne ha hvit header, men med grå bakgrunn.

Alternativt måtte vi ha gjort endel endringer i flere av malene, men da blir dette fort litt mye scope creep. Men vi kan gjerne ta en diskusjon i morgen!